Jenkinsfile

```diff
@@ -39,59 +39,22 @@ pipeline {
             }
         }
         stage('Build final version images') {
-            when {
-                expression {
-                    return isVersionTag(readCurrentTag())
-                }
-            }
             steps {
                 // TODO: remove build differences based on environments to allow reducing the
                 // number of different images. Maybe `demo` and `production` cold be enought
-                sh '''docker build \
-                    -t "fromdoppler/doppler-webapp:production-commit-${GIT_COMMIT}" \\
-                    --build-arg environment=production \\
-                    --build-arg react_app_router=browser \\
-                    --build-arg public_url="/" \\
-                    --build-arg version=production-${TAG_NAME}+${GIT_COMMIT} \\
-                    -f Dockerfile.swarm \\
-                    .'''
-                sh '''docker build \
-                    -t "fromdoppler/doppler-webapp:demo-commit-${GIT_COMMIT}" \\
-                    --build-arg environment=demo \\
-                    --build-arg react_app_router=browser \\
-                    --build-arg public_url="/" \\
-                    --build-arg version=demo-${TAG_NAME}+${GIT_COMMIT} \\
-                    -f Dockerfile.swarm \\
-                    .'''
                  sh '''docker build \
-                    -t "fromdoppler/doppler-webapp:int-commit-${GIT_COMMIT}" \\
+                    -t "local-doppler-webapp:int-commit-${GIT_COMMIT}" \\
                     --build-arg environment=int \\
                     --build-arg react_app_router=browser \\
                     --build-arg public_url="/" \\
-                    --build-arg version=int-${TAG_NAME}+${GIT_COMMIT} \\
-                    -f Dockerfile.swarm \\
-                    .'''
-                 sh '''docker build \
-                    -t "fromdoppler/doppler-webapp:qa-commit-${GIT_COMMIT}" \\
-                    --build-arg environment=qa \\
-                    --build-arg react_app_router=browser \\
-                    --build-arg public_url="/" \\
-                    --build-arg version=qa-${TAG_NAME}+${GIT_COMMIT} \\
+                    --build-arg version=int-v0.0.0+${GIT_COMMIT} \\
```

Comment:
What happend with _production_ and _demo_?

Reply:
It was a temporary change. Now, you can see all enviroments.